src/input/keyboard/keymap_file.rs

```diff
@@ -85,6 +85,12 @@ impl KeymapFile {
         Ok(())
     }
 
+    #[cfg(feature = "wayland_frontend")]
+    /// Send keymap string for processing
+    pub fn keymap_string(&self) -> String {
```

Comment:
What's the use case for this? And why do we clone the whole keymap string instead of just returning a reference to it?